packages/blade/rollup.config.mjs

```diff
@@ -186,21 +240,35 @@ const config = () => {
       .map((exportCategory) => [getCSSVariablesConfig({ exportCategory })])
       .flat();
   }
+
+  const components = glob.sync(`src/components/**/index.ts`);
```

Comment:
why not just `src/components/index.ts`?

Reply:
So basically this was the issue: https://github.com/rollup/rollup/issues/3916#issuecomment-751736158

Rollup was removing nested index.js files via treeshaking. 

But now that we have disabled treeshaking of modules on rollup side we don't need to add multiple entry point.

Reply:
aha yes I too faced this last time. Nice catch